src/jl_exported_funcs.inc

```diff
@@ -247,6 +247,7 @@
     XX(jl_get_UNAME) \
     XX(jl_get_world_counter) \
     XX(jl_get_zero_subnormals) \
+    XX(jl_get_num_stack_mappings) \
```

Comment:
Can you add a unit test of this function, to basically just test that it goes up if you spawn a task?

Reply:
I don't think it will always go up because it may just recycle and existing stack mapping when we spawn a new task.
`num_stack_mappings` has always been there so do you want to test that exporting is working well or that the existing metric is doing its job right?

Reply:
Eh, mainly i just wanted to exercise it, because it's useful to increase code coverage, incase of a silly typo somewhere or something. But this looks fine to me I agree. Most of these kinds of metrics aren't tested.

Reply:
nvm i just added it myself :) thanks adnan